packages/fuels-test-helpers/src/accounts.rs

```diff
@@ -85,6 +86,44 @@ pub async fn launch_custom_provider_and_get_wallets(
     Ok(wallets)
 }
 
+pub async fn connect_to_testnet_node_and_get_wallets(
+    num_wallets: usize,
+) -> Result<Vec<WalletUnlocked>> {
+    if num_wallets > 3 {
+        error!(
+            InvalidData,
+            "Trying to get more than 3 wallets from beta node"
+        );
+    }
+    let provider = Provider::connect(TESTNET_NODE_URL)
+        .await
+        .expect("Should be able to connect to {TESTNET_NODE_URL}");
```

Comment:
I suspect you wanted the value to be interpolated here:

```suggestion
        .unwrap_or_else(|_| panic!("Should be able to connect to {TESTNET_NODE_URL}"));
```

> [!NOTE]
> why not `expect(&format!(...))`? See [this clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#/expect_fun_call)